pkg/virtctl/guestfs/guestfs.go

```diff
@@ -378,15 +378,27 @@ func createLibguestfsPod(pvc, image, cmd string, args []string, kvm, isBlock boo
 	nonRoot := true
 	var uidRoot int64 = 0
 	var uid *int64
+	var containerSecurityContext *corev1.SecurityContext
 	if root {
 		nonRoot = false
 		uid = &uidRoot
+	} else {
```

Comment:
You might want to always do this or is there a reason why not?

Reply:
Nothig specific, I thought using root flag could also set leave more relaxed option. But actually it is just because the old images doesn't run with uid != 0. Let me always set them